edit.js

```diff
@@ -767,7 +765,6 @@ pe.addEventListener('packageremove', e => {
 let hoverTarget = null;
 let selectTarget = null;
 
-let transformControlsHovered = false;
```

Comment:
I'm pretty sure this will be needed when we do proper transform controls integration.